.github/workflows/versioning.yml

```diff
@@ -6,9 +6,9 @@ on:
 
 jobs:
   versioning:
-      if: "!contains(github.event.head_commit.message, 'Auto update version')"
-      runs-on: ubuntu-latest
-      steps:
+    if: github.repository != github.event.repository.full_name
```

Comment:
Should be `if: github.repository == github.event.repository.full_name` not `!=`.